src/Compression/CompressionCodecQuantile.cpp

```diff
@@ -0,0 +1,217 @@
+#ifdef USE_PCO
+#if USE_PCO
+
+#include "base/types.h"
+#include <Common/Exception.h>
+#include <Compression/ICompressionCodec.h>
+#include <Compression/CompressionFactory.h>
+#include <Compression/CompressionInfo.h>
+#include <DataTypes/IDataType.h>
+#include <Parsers/ASTLiteral.h>
+
+#include "config.h"
+
+#include <pco.h>
+
+namespace DB
+{
+
+class CompressionCodecQuantile : public ICompressionCodec
+{
+public:
```

Comment:
Could we mark the codec experimental for now (`ICodec::isExperimental()`)?

Reply:
done